src/queries.js

```diff
@@ -3,7 +3,7 @@ import debug from 'debug';
 import shallowEqual from 'buildo-state/lib/shallowEqual'; // TODO(split)
 import pick from 'lodash/pick';
 import _displayName from './displayName';
-import 'rxjs/add/operator/debounceTime';
+import { debounceTime } from 'rxjs/operators';
```

Comment:
you can read about `rxjs` changes here: https://www.academind.com/learn/javascript/rxjs-6-what-changed/

Reply:
This will probably require to have rxjs upgraded in `avenger` too. Otherwise we'll end up having two different versions of the lib (and it will probably break)

Reply:
it works with `"avenger": "^4.0.3"` but I agree that I don't like having two different versions of `rxjs`.. maybe the solution could be to drop `rxjs` in `react-avenger` (it is only used here I think...)?